developer_manual/client_apis/index.rst

```diff
@@ -7,11 +7,12 @@ Clients and Client APIs
 .. toctree::
     :maxdepth: 2
 
-    general
```

Comment:
This was sorted by priority and I think having general first would still be helpful, although I get the alphabetic one.
So maybe alphabetic with the exception of general?

Reply:
Yes :+1:

Reply:
Done